Makefile

```diff
@@ -6,7 +6,10 @@ venv:
 install: venv
 	. venv/bin/activate; pip install .
 
-test-install: install
+pre-test-install:
+	@`python --version` || (echo "Python is not installed, please install Python 2 or Python 3"; exit 1)
```

Comment:
Yeah, I was wondering whether we should have it on `install` or `test-install`, or both..?